src/dbt_client/dbtCoreIntegration.ts

```diff
@@ -85,6 +86,32 @@ export class DBTCoreProjectDetection
     private executionInfrastructure: DBTCommandExecutionInfrastructure,
   ) {}
 
+  private getPackageInstallPathFallback(projectDirectory: Uri): string {
+    const dbtProjectFile = path.join(
+      projectDirectory.fsPath,
+      "dbt_project.yml",
+    );
+    if (existsSync(dbtProjectFile)) {
+      const dbtProjectConfig: any = parse(readFileSync(dbtProjectFile, "utf8"));
+      const packagesInstallPath = dbtProjectConfig["packages-install-path"];
+      if (packagesInstallPath) {
+        let absolutePath;
```

Comment:
Maybe you can name this one packageInstallPath and above packageInstallPathFromYaml.

Anyway absolutepath is a bit of a generic name.